.gitignore

```diff
@@ -1,6 +1,8 @@
 _bin/
 _output/
 .python_virtual_env
+/yarn.lock
+/node_modules
```

Comment:
Since `node_modules` is a directory, I suggest to modify this line slightly as follows:
→ `node_modules/`

Reply:
^